src/Stories/our-journey-of-transforming-to-a-devops-culture.md

```diff
@@ -31,16 +36,53 @@
 // edward
 ###Evolution
 // edward
+
 ##What's a ...
-//?
+
```

Comment:
This section nicely summarizes the problem at hand that we were trying to solve.